public/app/plugins/panel/xychart/config.ts

```diff
@@ -36,13 +36,6 @@ export function getScatterFieldConfig(cfg: FieldConfig): SetFieldConfigOptionsAr
         hideFromDefaults: true,
       },
 
-      [FieldConfigProperty.Thresholds]: {
-        hideFromDefaults: true,
-      },
-      [FieldConfigProperty.Mappings]: {
-        hideFromDefaults: true,
-      },
-
```

Comment:
keeping this available in general options is helpful since it doesnt require going into overrides for specific color field.